scripts/run.R

```diff
@@ -26,10 +26,9 @@ suppressPackageStartupMessages(source(here::here("R", "load_all.R")))
 #   )
 #   # Save to disk
 #   saveRDS(scorecards, "exploration-scorecards-2023-10-04.RDS")
-print(glue::glue("starting a run at {Sys.time()}"))
```

Comment:
why drop the time information?

Reply:
cause targets with timestamp handles that